examples/advanced/cross-validation-without-training/README.md

```diff
@@ -0,0 +1,31 @@
+# Cifar10_fedavg example re-run cross site validation using the previous trained results
+
+## Introduction
+
+This example shows how to re-run the NVFlare cross-site validation without the training workflow, making use of the previous run results. The example uses the cifar10_fedavg NVFlare job configuration.
+
+### How is the example built
+
+This example uses the exact same NVFlare job definition of cifar10_fedavg, which contains the training workflow, followed by the cross-site validation workflow. These are the steps of changes made to enable re-run the cross-site validation without the training.
+
+1. removed the scatter_gather_ctl workflow from the config_fed_server.json
+
+2.  In the PTFileModelLocator, add the following lines to initialize and create the persistence_manager instance in the model_persister. This is a NVFlare core codes change.
```

Comment:
I think we should remove this step since this will be included in our code.

Reply:
Correct. removed in README.